recipes/zlib/1.2.11/conanfile.py

```diff
@@ -1,22 +1,29 @@
-import os
 from conans import ConanFile, tools, CMake
-from conans.errors import ConanException
+import os
+
+required_conan_version = ">=1.43.0"
 
 
 class ZlibConan(ConanFile):
     name = "zlib"
-    version = "1.2.11"
```

Comment:
The directory is called 1.2.11. I would either change the directory name as well or leave this here

Reply:
I can rename the directoy to all, then merge 1.2.8 in this one.

Reply:
done, the diff is still understandable